js/repl/types.js

```diff
@@ -141,3 +141,5 @@ export type Transition = {
   visitorType: string,
   currentNode?: string,
 };
+
+export type SupportedFileExtension = ".js" | ".jsx" | ".ts" | ".tsx";
```

Comment:
Do we need `.mjs` when `config.sourceType === "module"`?

Reply:
That would be more consistent with the Node.js experience, I would say yes.

Reply:
Well, it isn't observable to users.

Reply:
True. Given that we don't have `.mjs` specific transforms in the preset, we don't have to add it here.